submissions/2021-11-08-QMDataset-Solvated-Amino-Acids-single-points/Dataset_Generation.ipynb

```diff
@@ -32,7 +32,7 @@
     "                                                                   maxiter=200, \n",
     "                                                                   scf_properties=[SCFProperties.Dipole, SCFProperties.Quadrupole, SCFProperties.WibergLowdinIndices, \n",
```

Comment:
I don't see replacement of `v1.0` with `v1.1` among the notebook contents, and I don't see it in the `dataset.json.bz2` either. We'll need this to avoid confusion with #239.

Reply:
this one never got computed, failed during qca submission stage due to mishandling of multiple-molecules, are the records created for previous submission and this should be v1.1?

Reply:
This dataset exists already on public QCA, and it appears to have records. So we'll need this one to be v1.1 unfortunately.